examples/booking_app/lib/feature_hotel/search/views/search_page.dart

```diff
@@ -170,4 +145,198 @@ class _SearchPageState extends State<SearchPage> with TickerProviderStateMixin {
       ),
     );
   }
+
+  Widget _buildFilters(BuildContext context) => Padding(
```

Comment:
@DDavidPrime can we break down this huge method to smaller once. In that way we would improve the readability. :)

Reply:
Improved by separating its contents into two separate parts: one for each filter.